Vagrantfile

```diff
@@ -11,15 +11,15 @@ Vagrant.configure("2") do |config|
   # "Fix" for error 'stdin: is not a tty' (see https://github.com/mitchellh/vagrant/issues/1673)
   config.vm.provision :shell, inline: 'apt-get purge -y chef*'
   config.vm.provision :shell, inline: 'sed -i \'s/^mesg n$/tty -s \&\& mesg n/g\' /root/.profile'
- 
+
   config.vm.network :forwarded_port, guest: 3006, host: 3006
 
   config.vm.provision "docker" do |d|
-    d.pull_images "quay.io/coreos/etcd"
-    d.run "quay.io/coreos/etcd", args: "-p 4001:4001", demonize: true
+    d.pull_images "quay.io/coreos/etcd:v0.4.6"
```

Comment:
We are currently running different version in prod, i'm not sure if you want to align it with that?